cms/envs/common.py

```diff
@@ -60,6 +60,8 @@
 
     # Django REST framework configuration
     REST_FRAMEWORK,
+
+    STATICI18N_OUTPUT_DIR
```

Comment:
Unfortunately, I don't think so, because that's a setting defined by the package itself: http://django-statici18n.readthedocs.org/en/latest/settings.html